src/python/gudhi/alpha_complex.pyx

```diff
@@ -55,39 +57,56 @@ cdef class AlphaComplex:
     cdef Alpha_complex_interface * this_ptr
 
     # Fake constructor that does nothing but documenting the constructor
-    def __init__(self, points=None, off_file='', precision='safe'):
+    def __init__(self, points=[], off_file='', weights=[], precision='safe'):
```

Comment:
I think `weights=None` is a bit clearer than an empty list, when we want the unweighted version.

Reply:
You are right, it is clearer from a user interface point of view. Done on afd0a890